### PR TITLE
Map parameters of commands

### PR DESCRIPTION
**Pull request by apple502j**
Mapped lambda parameters (and some methods) of commands.

### CODE REVIEW

File: mappings/net/minecraft/server/command/ExperienceCommand.mapping [hunk 1]
```diff
@@ -26,3 +46,13 @@ CLASS net/minecraft/class_3054 net/minecraft/server/command/ExperienceCommand
...
+		METHOD method_13340 (Lnet/minecraft/class_3222;)I
+			ARG 0 player
+		METHOD method_13341 (Lnet/minecraft/class_3222;)I
+			ARG 0 player
+		METHOD method_13342 (Lnet/minecraft/class_3222;Ljava/lang/Integer;)Z
+			ARG 0 player
+			ARG 1 level
+		METHOD method_13343 (Lnet/minecraft/class_3222;Ljava/lang/Integer;)Z
+			ARG 0 player
+			ARG 1 points
```
Comment (liach) [inline mappings/net/minecraft/server/command/ExperienceCommand.mapping]: Maybe just `xp`?

File: mappings/net/minecraft/server/command/LootCommand.mapping [hunk 2]
```diff
@@ -46,26 +70,54 @@ CLASS net/minecraft/class_3039 net/minecraft/server/command/LootCommand
...
+	METHOD method_13214 (Lcom/mojang/brigadier/context/CommandContext;Ljava/util/List;Lnet/minecraft/class_3039$class_3040;)I
+		ARG 0 context
+		ARG 1 stacks
+		ARG 2 messageSender
+	METHOD method_13215 (Lnet/minecraft/class_3039$class_3041;Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 1 context
+	METHOD method_13216 (Lnet/minecraft/class_3039$class_3041;Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 1 context
+	METHOD method_13217 (Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
+		ARG 0 entituName
```
Comment (liach) [inline mappings/net/minecraft/server/command/LootCommand.mapping]:
```suggestion
		ARG 0 entityName
```

File: mappings/net/minecraft/server/command/ReloadCommand.mapping [hunk 3]
```diff
@@ -2,3 +2,16 @@ CLASS net/minecraft/class_3097 net/minecraft/server/command/ReloadCommand
 	FIELD field_25343 LOGGER Lorg/apache/logging/log4j/Logger;
 	METHOD method_13529 register (Lcom/mojang/brigadier/CommandDispatcher;)V
 		ARG 0 dispatcher
+	METHOD method_13530 (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 context
+	METHOD method_13531 (Lnet/minecraft/class_2168;)Z
+		ARG 0 source
+	METHOD method_29478 getNewDatapacks (Lnet/minecraft/class_3283;Lnet/minecraft/class_5219;Ljava/util/Collection;)Ljava/util/Collection;
```
Comment (liach) [inline mappings/net/minecraft/server/command/ReloadCommand.mapping]:
```suggestion
	METHOD method_29478 findNewDataPacks (Lnet/minecraft/class_3283;Lnet/minecraft/class_5219;Ljava/util/Collection;)Ljava/util/Collection;
```

File: mappings/net/minecraft/server/command/ReloadCommand.mapping [hunk 4]
```diff
@@ -2,3 +2,16 @@ CLASS net/minecraft/class_3097 net/minecraft/server/command/ReloadCommand
...
+		ARG 0 resourcePackManager
```
Comment (liach) [inline mappings/net/minecraft/server/command/ReloadCommand.mapping]:
```suggestion
		ARG 0 dataPackManager
```
though the type is `ResourcePackManager`, I prefer a name that fit the context instead.

File: mappings/net/minecraft/server/command/ReloadCommand.mapping [hunk 5]
```diff
@@ -2,3 +2,16 @@ CLASS net/minecraft/class_3097 net/minecraft/server/command/ReloadCommand
...
+		ARG 1 saveProperties
+		ARG 2 enabledDatapacks
```
Comment (liach) [inline mappings/net/minecraft/server/command/ReloadCommand.mapping]:
```suggestion
		ARG 2 enabledDataPacks
```
Just fyi, this is how we capitalize "data pack". It's two words, like "item stack".

File: mappings/net/minecraft/server/command/ReloadCommand.mapping [hunk 6]
```diff
@@ -2,3 +2,16 @@ CLASS net/minecraft/class_3097 net/minecraft/server/command/ReloadCommand
...
+	METHOD method_29479 (Lnet/minecraft/class_2168;Ljava/lang/Throwable;)Ljava/lang/Void;
+		ARG 1 throwable
+	METHOD method_29480 tryReloadDatapacks (Ljava/util/Collection;Lnet/minecraft/class_2168;)V
+		ARG 0 datapacks
```
Comment (liach) [inline mappings/net/minecraft/server/command/ReloadCommand.mapping]:
```suggestion
	METHOD method_29480 tryReloadDataPacks (Ljava/util/Collection;Lnet/minecraft/class_2168;)V
		ARG 0 dataPacks
```

**Review by liach — APPROVED** [review]
Thank you!

**Review by enbrain — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]
Looks good. Thanks for this 👍

**Pull request merged**